### PR TITLE
top: update page

### PR DESCRIPTION
**Pull request by einverne**
- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by Aracki** [comment]
Is it possible to achieve this on other platforms? @einverne

**Comment by einverne** [comment]
> Is it possible to achieve this on other platforms? @einverne

Sorry @Aracki , I don't have a mac, so I cannot test it. We can keep watch, and let some work under mac to verify it.

**Comment by Aracki** [comment]
I have tried with osx and it's not working. I was wondering what about windows.

File: pages/linux/top.md [hunk 1]
```diff
@@ -14,6 +14,10 @@
 
 `top -u {{user_name}}`
 
+- Show individual threads of given process:
```
Comment (zdroid) [inline pages/linux/top.md]:
```suggestion
- Show the individual threads of a given process:
```

**Comment by sbrl** [comment]
Hrm, if it doesn't work on macOS then we may have to have a macOS-specific copy of the page with the  example removed.

**Comment by mebeim** [comment]
Not sure what's the problem here, the page is already under `linux`, the fact that this does not work on macOS is not a problem, there already is [`pages/osx/top.md`](https://github.com/tldr-pages/tldr/blob/master/pages/osx/top.md). LGTM.

**Comment by sbrl** [comment]
Ah, I wasn't aware that we already had a macOS-specific version of the page. That makes it all ok then :P

**Pull request merged**